src/xcode/ENA/ENA/Source/Developer Menu/Features/DMWarnOthersNotificationViewController.swift

```diff
@@ -0,0 +1,198 @@
+//
+// Corona-Warn-App
+//
+// SAP SE and all other contributors
+// copyright owners license this file to you under the Apache
+// License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+#if !RELEASE
+
+import UIKit
+
+final class DMWarnOthersNotificationViewController: UIViewController, UITextFieldDelegate {
+	
+	// MARK: Properties
+	private var time1Textfield: UITextField!
+	private var time2Textfield: UITextField!
+	
+	private var time1Label: UILabel!
+	private var time2Label: UILabel!
+	
+	private let store: Store
+	private var warnOthers: OthersWarnable
+	
+	init(warnOthers: OthersWarnable, store: Store) {
+		self.store = store
+		self.warnOthers = warnOthers
+		super.init(nibName: nil, bundle: nil)
+	}
+	
+	@available(*, unavailable)
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+	
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		
+		view.backgroundColor = .systemBackground
+		
+		let titleLabel = UILabel(frame: .zero)
+		titleLabel.translatesAutoresizingMaskIntoConstraints = false
+		titleLabel.numberOfLines = 0
+		titleLabel.text = "Warn others notification settings"
+		titleLabel.font = UIFont.enaFont(for: .headline)
+		
+		let descriptionLabel = UILabel(frame: .zero)
+		descriptionLabel.translatesAutoresizingMaskIntoConstraints = false
+		descriptionLabel.numberOfLines = 0
+		descriptionLabel.text = "You can use the two input fields below to maintain the time until the notifications will be shown.\nTime is maintained in seconds."
+		descriptionLabel.font = UIFont.enaFont(for: .subheadline)
+		
+		let saveButton = UIButton(frame: .zero)
+		saveButton.translatesAutoresizingMaskIntoConstraints = false
+		saveButton.setTitle("Save timer values", for: .normal)
+		saveButton.addTarget(self, action: #selector(scheduleNotificationsButtonTapped), for: .touchUpInside)
+		saveButton.setTitleColor(.enaColor(for: .buttonPrimary), for: .normal)
+		
+		let resetDefaultsButton = UIButton(frame: .zero)
+		resetDefaultsButton.translatesAutoresizingMaskIntoConstraints = false
+		resetDefaultsButton.setTitle("Reset values to default", for: .normal)
+		resetDefaultsButton.addTarget(self, action: #selector(resetDefaultsButtonTapped), for: .touchUpInside)
+		resetDefaultsButton.setTitleColor(.enaColor(for: .buttonDestructive), for: .normal)
+		
+		let descriptionLabelResetNotifications = UILabel(frame: .zero)
```

Comment:
`Label` should be last, so could be `resetNotificationsDescriptionLabel`

Reply:
Done